python/test/test_property_graph.py

```diff
@@ -52,17 +50,41 @@ def test_get_edge_dest(graph):
     assert graph.get_edge_dest(1) == 8014
 
 
+def test_edge_data_frame(graph):
+    edges = graph.edges()
+    assert len(edges.columns) == 18 + 3
+    assert len(edges) == 43072
+    assert edges.at[0, "dest"] == 8014
+    assert edges["dest"][1] == 8014
```

Comment:
Can we also get a test asserting the value of edges["src"]? Would be useful as an example seeing how to access all the columns of the DataFrame.

Reply:
Tests != documentation. I'll put an example in the doc string.